### PR TITLE
Live log

### LINKED ISSUE #1211: WebUI: build link from monitor no longer points to live log
In monitor view of the project, links are pointing to raw log instead of live build log.

Like: https://build.opensuse.org/build/openSUSE:Factory:PowerPC/standard/ppc64le/MozillaFirefox/_log
instead of 
https://build.opensuse.org/package/live_build_log/openSUSE:Factory:PowerPC/MozillaFirefox/standard/ppc64le

### PR DESCRIPTION
**Pull request by ChrisBr**
If the package is not local we showed the raw log instead. This commit will show the livelog instead.
Fix #1211.

### CODE REVIEW

**Comment by ChrisBr** [comment]
@adrianschroeter please review!

**Comment by ChrisBr** [comment]
@adrianschroeter added additional tests!

File: src/api/app/controllers/webui/package_controller.rb [hunk 1]
```diff
@@ -813,11 +813,18 @@ def set_job_status
 
   def live_build_log
     required_parameters :arch, :repository
-    if @package and not @package.check_source_access?
+
+    @project = Project.find_by(name: params[:project])
+    @project = Project.find_remote_project(params[:project]).first unless @project
```
Comment (hennevogel) [inline src/api/app/controllers/webui/package_controller.rb]: Why not get_by_name?

Comment (ChrisBr) [inline src/api/app/controllers/webui/package_controller.rb]: Hm, good point!

**Comment by adrianschroeter** [comment]
please rebase :)

**Comment by ChrisBr** [comment]
@adrianschroeter done!

**Comment by mdeniz** [comment]
LGTM

**Pull request merged**